Index: goodscenter/src/main/java/cn/xwj/goods/di/component/CartComponent.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- goodscenter/src/main/java/cn/xwj/goods/di/component/CartComponent.kt	(date 1528210895029)
+++ goodscenter/src/main/java/cn/xwj/goods/di/component/CartComponent.kt	(date 1528210895029)
@@ -0,0 +1,18 @@
+package cn.xwj.goods.di.component
+
+import cn.xwj.baselibrary.di.component.ActivityComponent
+import cn.xwj.baselibrary.di.scope.PerComponentScope
+import cn.xwj.goods.di.module.CartModule
+import dagger.Component
+
+/**
+ * Author: xw
+ * Email:i.xiaowujiang@gmail.com
+ * Date: 2018-06-03 2018/6/3
+ * Description: CartComponent
+ */
+@PerComponentScope
+@Component(dependencies = [(ActivityComponent::class)],modules = [(CartModule::class)])
+interface CartComponent {
+
+}
